### ISSUE TITLE
Tagging releases

### ISSUE THREAD

**Issue by alucryd** [Nanook/NKitv1#1]
Hi there,

Thank you for all this work, it's pretty awesome!

I'm in the process of creating a package for Arch Linux, but I'd need a tagged release to do it the proper way, would that be much trouble to create one?

Also I noticed the binaries still print version 1.3 when gbatemp and the readme advertise 1.4. Is this repo up to date?

Cheers.

**Comment by farouk327** [comment]
@alucryd
Yes the binaries  still in  1.3 i don't  know why i try  to compiling it to run  property in windows 7 1sp 64bit but still missing some codes & things we need the dev help😝

**Comment by Nanook** [comment]
I'll address this :)

**Comment by keks24** [comment]
@alucryd 

I forked the repository and provided one release. I am actually not sure, if it is version `1.3` or `1.4`. Also I am not going to maintain this fork, therefore it is archived. :)

https://github.com/keks24/NKit/releases/tag/v1.4

**Comment by alucryd** [comment]
@keks24 Thanks, but I've lost interest in NKIT and am now focusing on RVZ from the dolphin emulator.
